qutebrowser/config/configinit.py

```diff
@@ -90,9 +90,6 @@ def _init_envvars():
     if config.val.qt.force_platform is not None:
         os.environ['QT_QPA_PLATFORM'] = config.val.qt.force_platform
 
-    if config.val.window.hide_wayland_decoration:
-        os.environ['QT_WAYLAND_DISABLE_WINDOWDECORATION'] = '1'
-
```

Comment:
You'll still need to do this (but with the new setting name), otherwise the window decoration shows up again on wayland.